internal/commands/cnab_test.go

```diff
@@ -0,0 +1,87 @@
+package commands
+
+import (
+	"testing"
+
+	"github.com/docker/cli/cli/command"
+	"gotest.tools/assert"
+)
+
+func TestRequiresBindMount(t *testing.T) {
+	dockerCli, err := command.NewDockerCli()
+	assert.NilError(t, err)
+
+	testCases := []struct {
+		name               string
+		targetContextName  string
+		targetOrchestrator string
+		expectedRequired   bool
+		expectedEndpoint   string
+		expectedError      string
+	}{
+		{
+			name:               "kubernetes-orchestrator",
+			targetContextName:  "target-context",
```

Comment:
I'm unsure what populates this context (or maybe I misunderstand how this works), I can't find it here and my `grep` in the existing source tree isn't showing it. In any case do we not need to be sure we test at least `unix://` and `tcp://` as well as perhaps some invalid syntax?

Reply:
These tests only check for the 2 hardcoded conditions (kubernetes target and default context). I've split out the socket path resolution into a helper method and added some tests for that. The e2e tests should ensure the rest works for both unix:// and tcp:// (@simonferquel please can you confirm this?)

Reply:
Good enough for me, thanks!